kernel/ixgbe/src/lib.rs

```diff
@@ -256,7 +255,7 @@ impl IxgbeNic {
         if (link_speed == LinkSpeedMbps::LSUnknown) | (link_speed == LinkSpeedMbps::LS100) {
             return Err("Ixgbe driver can only be configured for 1 Gbps or 10 Gbps link speeds");
         }
-        if (virtualization_enabled & interrupts.is_some()) | (virtualization_enabled & rss_enabled) {
+        if (enable_virtualization & interrupts.is_some()) | (enable_virtualization & enable_rss) {
```

Comment:
I think this should be a logical boolean AND (`&&`) instead of a bitwise operator (`&`), right?